sway/desktop/output.c

```diff
@@ -765,6 +806,8 @@ static void handle_present(struct wl_listener *listener, void *data) {
 		return;
 	}
 
+	output->refresh_nsec = output_event->refresh;
```

Comment:
Where exactly should it be zeroed? Along with the `struct timespec last_presentation` that I'm adding.

Reply:
Right after we render or scan-out (`wlr_output_commit`), probably?

Reply:
Actually, looks like there's no need to zero them. They provide valid "last known" presentation data. If it's out of date (for example, if sway running as a wayland application was hidden and hasn't received presentation feedback in a while) then the delay computation will simply fall back to no delay, whereas some other part of the code, potentially interested in the refresh rate, could still use the value, which likely hasn't changed since the last time sway received presentation feedback.